lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -108,12 +123,20 @@ export class RefreshTokenClient extends BaseClient {
      */
     private async acquireTokenWithCachedRefreshToken(request: CommonSilentFlowRequest, foci: boolean) {
         // fetches family RT or application RT based on FOCI value
+
+        // @ts-ignore
+        const atsMeasurement = this.performanceClient?.startMeasurement("RefreshTokenClientAcquireTokenWithCachedRefreshToken", request.correlationId);
         const refreshToken = this.cacheManager.readRefreshTokenFromCache(this.config.authOptions.clientId, request.account, foci);
 
-        // no refresh Token
         if (!refreshToken) {
+            atsMeasurement?.discardMeasurement();
             throw InteractionRequiredAuthError.createNoTokensFoundError();
         }
+        // attach cached RT size to the current measurement
+        atsMeasurement?.endMeasurement({
+            success: true,
+            refreshTokenSize: refreshToken.secret.length
```

Comment:
Is this measuring the RT size a 2nd time?

Reply:
Should we measure the RT size at time of acquisition rather than at time of use?

Reply:
Thanks for pointing this, great catch! Have removed this as its redundant, only measuring at time of RT acquistion in acquire token()